src/StaticWebAssetsSdk/Targets/Microsoft.NET.Sdk.StaticWebAssets.HtmlImportMap.targets

```diff
@@ -0,0 +1,81 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.StaticWebAssets.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+
+<Project ToolsVersion="14.0">
+    
+  <UsingTask TaskName="Microsoft.AspNetCore.StaticWebAssets.Tasks.WriteImportMapToHtml" AssemblyFile="$(StaticWebAssetsSdkBuildTasksAssembly)" />
+
+    <PropertyGroup>
+    <ResolveBuildRelatedStaticWebAssetsDependsOn>
+      $(ResolveStaticWebAssetsInputsDependsOn);
+      _UpdateHtmlFiles;
+    </ResolveBuildRelatedStaticWebAssetsDependsOn>
+    <ResolveCompressedFilesDependsOn>
+      $(ResolveCompressedFilesDependsOn);
+      _UpdateHtmlFiles
+    </ResolveCompressedFilesDependsOn>
+  </PropertyGroup>
+  <ItemGroup>
+    <StaticWebAssetFingerprintPattern Include="Js" Pattern="*.js" Expression="#[.{fingerprint}]!" />
```

Comment:
I don't think it's correct for us to do this, since importmap only applies to ES6 modules, not regular scripts.

Reply:
Yeah. We leave this as optional step to the users. It only matters in case of `wasmbrowser` template. In blazor we should do hard fingerprint on blazor.wasm.js if the importmap generation is enabled and we are in standalone mode, right?